mcc/odin/odin/snapshot_list_test.go

```diff
@@ -2,25 +2,41 @@ package odin_test
 
 import (
 	"testing"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/rds"
 
 	"github.com/poka-yoke/spaceflight/mcc/odin/odin"
 )
 
+const (
+	RFC8601 = "2006-01-02T15:04:05+07:00"
```

Comment:
As mentioned before, this is the format AWS specifies for this field.
As response to what @dsager asked about not being a more typical `%d-%m-%YT%H:%M:%S%Z` format or similar, in Go date and time formatting is taken from a specific format for the date and time Jan 2nd, 2006, at 15:04:05 in GMT+7, which is represented by something called magic datetime (1 2 3 4 5 6 7), where they represent month, day, hour, minute, second, year after 2000, and timezone. More: https://golang.org/pkg/time/#pkg-constants